cumulusci/tasks/salesforce/ActivateProcessBuilderProcesses.py

```diff
@@ -0,0 +1,36 @@
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+
+
+class ActivateProcessBuilderProcesses(BaseSalesforceApiTask):
+    """
+    Create a Salesforce Activation Process Builder which takes in a list as options and
+    runs flows only on those listed DeveloperName's.
+    """
+
+    def _run_task(self):
+        self.logger.info(
+            f'Allowing only rows with the following DeveloperNames {self.options.get("developer-names")}'
+        )
+        self.logger.info("Querying flow definitions...")
+        res = self.tooling.query(
+            "SELECT Id,ActiveVersion.VersionNumber,LatestVersion.VersionNumber,DeveloperName FROM FlowDefinition"
+        )
+        for listed_flow in res["records"]:
+            if listed_flow["DeveloperName"] in self.options.get("developer-names"):
+                self.logger.info(f'Processing: {listed_flow["DeveloperName"]}')
+                path = "tooling/sobjects/FlowDefinition/{0}".format(listed_flow["Id"])
+                urlpath = self.sf.base_url + path
+                data = {
+                    "Metadata": {
+                        "activeVersionNumber": listed_flow["LatestVersion"][
+                            "VersionNumber"
+                        ]
+                    }
+                }
+
+                response = self.tooling._call_salesforce("PATCH", urlpath, json=data)
+                self.logger.info(response)
+            else:
+                self.logger.info(
+                    f'Skipping DeveloperName: {listed_flow["DeveloperName"]} not in given developer-names list.'
```

Comment:
I think it's safe to omit this branch. Some orgs may have a _lot_ of Processes and versions, so this could put a lot of chatter into the log.

Reply:
Im not following do you mean continue in the else statement?

Reply:
No, I just mean remove the `else:` branch entirely here - I just don't think we need to do this log.